File: comparativa/ejemplos/07-clases/person.py
import sys

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def greet(self):
        print "Hello! I am " + self.name

    def __str__(self):
        return "Name: " + self.name + ", Age: " + str(self.age)

p = Person('Tomas', 44)
p.greet
print p

File: comparativa/ejemplos/07-clases/person.rb
class Person
  def initialize(name, age)
    @name = name
    @age = age
  end

  def greet
    puts("Hello! I'm #{@name}.")
  end

  def to_s
    "Name: #{@name}, Age: #{@age}"
  end
end

p = Person.new('Tomas', 44)
p.greet
puts p
